fix/upp/postxconfig-NT-fv3lam_rrfs.txt

```diff
@@ -10235,7 +10235,7 @@ ACM_100YARI_EXCEEDANCE
 ?
 1
 tmpl4_8
-DEPWSS
+GWLOWS
```

Comment:
Should only change one line but why so many GWLOWS changes.

Reply:
I made all the average recurrence interval exceedances the same GRIB2 name for now, until we get an established GRIB2 name for them.